janus-idp/operator
# Integration tests

## Description
Added the rest of controller-tests cases to integration-tests and removed controller-tests

## Which issue(s) does this PR fix or relate to

https://github.com/janus-idp/operator/issues/120

## PR acceptance criteria


## How to test changes / Special notes to the reviewer
To test locally:
`make test `
To test on openshift make sure your current context is pointed to working OS cluster and run:
`make integration-test USE_EXISTING_CLUSTER=true`
